crates/libs/windows/Cargo.toml

```diff
@@ -14,6 +14,7 @@ rust-version = "1.64"
 [package.metadata.docs.rs]
 default-target = "x86_64-pc-windows-msvc"
 targets = []
+rustc-args = ["--cfg", "docsrs"]
```

Comment:
`rustc-args` don't apply to the doc-generation step but to the check/proc-macro builds of all the dependencies. This should be `rustdoc-args`.

Reply:
Yeah, I just figured that. Btw, maybe the [cross-compiling](https://docs.rs/about/builds#cross-compiling) section should mention this?

Reply:
Yep https://github.com/rust-lang/docs.rs/issues/2284

Reply:
Thanks!